packages/battery/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+* Update lower bound of dart dependency to 2.1.0.
```

Comment:
what is the rationale for this bump?

Reply:
the `pub publish` check started to fail because the plugin depends on a pre-release version of dart.